source/manual/alerts/content-publisher-healthcheck-not-okay.html.md

```diff
@@ -0,0 +1,29 @@
+---
+owner_slack: "#govuk-pubworkflow-dev"
+title: content-publisher app healthcheck not ok
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-02-24
+review_in: 6 months
+---
+
+There are a few things that this alert can be, you'll need to click on the alert to find out more details on what's gone wrong. There are more details and common fixes listed below.
+
+## Government Data Check
+
+This means that Content Publisher is having trouble updating government data. Ordinarily it does this every fifteen minutes and seeing this error means it hasn't happened in at least 6 hours. After 24 hours the cache of government data will clear and the app will stop working as expected. The following suggestions should help to isolate the problem.
```

Comment:
I'd suggest adding some information (just one sentence should suffice) about what this data is, and where it is updated from.  This should help any developers who don't have any context about Content Publisher.

Reply:
Nice one, have adjusted existing sentences to give a bit more context :)